storage/src/mmr/iterator.rs

```diff
@@ -29,6 +29,36 @@ impl PeakIterator {
             two_h,
         }
     }
+
+    /// Return if an MMR of the given `size` has a valid structure.
```

Comment:
nit: Might want to add a tad more (under this line) about how this is actually done.

Reply:
added